resources/docs/livewire/creating-chirps.md

```diff
@@ -0,0 +1,504 @@
+| contains the "web" middleware group. Now create something great!
+|
+*/
+
+Route::get('/', function () {
+    return view('welcome');
+});
+// [tl! collapse:end]
+Route::get('/dashboard', function () {
+    return view('dashboard');
+})->middleware(['auth', 'verified'])->name('dashboard');
+// [tl! add:start]
+Route::get('/chirps', [ChirpController::class, 'index'])
+    ->middleware(['auth', 'verified'])
+    ->name('chirps'); // [tl! add:end]
+
+require __DIR__.'/auth.php';
+```
+
+This will create the following route:
+
+Verb      | URI                    | Action | Route Name
+----------|------------------------|--------|---------------------
+GET       | `/chirps`              | index  | chirps
+
+> **Note**
+> You may view all of the routes for your application by running the `php artisan route:list` command.
+
+Let's test our route and controller by returning a test message from the `index` method of our new `ChirpController` class:
+
+```php filename=app/Http/Controllers/ChirpController.php
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request; // [tl! remove]
+use Illuminate\Http\Response; // [tl! add]
+
+class ChirpController extends Controller
+{
+    /**
+     * Display a listing of the resource. [tl! add:-1,7]
+     */
+    public function index(): Response
+    {
+        return response('Hello, World!');
+    }
+}
+```
+
+If you are still logged in from earlier, you should see your message when navigating to [http://localhost:8000/chirps](http://localhost:8000/chirps), or [http://localhost/chirps](http://localhost/chirps) if you're using Sail!
+
+## Livewire
+
+Not impressed yet? Let's update the `index` method of our `ChirpController` class to render a Blade view:
+
+```php filename=app/Http/Controllers/ChirpController.php
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Response;// [tl! remove]
+use Illuminate\View\View;// [tl! add]
+
+class ChirpController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(): Response// [tl! remove]
+    public function index(): View// [tl! add]
+    {
+        return response('Hello, World!');// [tl! remove]
+        return view('chirps', [// [tl! add:start]
+            //
+        ]);// [tl! add:end]
+    }
+}
+```
+
+We can then create our Blade view template with a Livewire component with a form for creating new Chirps:
+
+```blade filename=resources/views/chirps.blade.php
+<x-app-layout>
+    <div class="max-w-2xl mx-auto p-4 sm:p-6 lg:p-8">
+        <livewire:chirps.create />
+    </div>
+</x-app-layout>
+```
+
+And finally, we can create a Livewire component to render the form. For that, you may use the `make:volt` Artisan command.
+
+Note that the snippet below offers two different ways to create the component: one using a `Class` API and the other using a `Functional` API. You will see both APIs throughout this tutorial, and you can choose the one you prefer:
+
+```shell tab=Class
```

Comment:
It would be cool to add icons for the tabs as we have for Vue and React. Not sure what they should be though. Maybe the Volt logo? Not sure if there is anything we could do to differentiate the two APIs though.

Reply:
I also wonder whether the Functional API should be the first tab, as it seems to be the "default" approach in the Volt docs and `make:volt` command.

Reply:
<img width="333" alt="Screenshot 2023-09-06 at 16 24 30" src="https://github.com/laravel/bootcamp.laravel.com/assets/5457236/5876db4c-c243-407b-9499-07a96171426f">